lib/cluster-providers/ec2-cluster-provider.ts

```diff
@@ -84,8 +84,15 @@ export class EC2ClusterProvider implements ClusterProvider {
     readonly options: EC2ProviderClusterProps;
 
 
-    constructor(options?: EC2ProviderClusterProps) {
-        this.options = options ?? { version: KubernetesVersion.V1_19 };
+    constructor(scope?: Construct, options?: EC2ProviderClusterProps) {
+        if (options != null)
+            this.options = options;
+        else {
```

Comment:
we consistently use {} with all if/else. 
In this case I think you could safely leverage tripod operator, e.g. 

this.options = scope ?  {a} : {b};